datarobot_provider/hooks/credentials.py

```diff
@@ -61,7 +62,7 @@ def get_conn(self) -> Credential:
         # Trying to find existing DataRobot Credentials managed by Airflow provider:
```

Comment:
While you are in this method, can you rename it please `get_conn` is ambiguous and confusing with how it differs from `self.get_connection`. Maybe it should be called `get_or_create_credential`?

Reply:
get_conn is implementation of get_conn abstract method from BaseHook:     
```
def get_conn(self) -> Any:
        """Returns connection for the hook."""
        raise NotImplementedError()
```

Reply:
Propose for  get_connection is different is just fetching stored values from secret storage
    def get_connection(cls, conn_id: str) -> Connection:
        conn = Connection.get_connection_from_secrets(conn_id)

Reply:
Ok, then if we are stuck with `get_conn`, I feel we should rename `get_connection`

Reply:
get_connection is also BaseHook method, so we cant rename it, its core class of Airflow

Reply:
Ah ok, that's unfortunate then